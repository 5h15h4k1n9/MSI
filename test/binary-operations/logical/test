true && false
true || false
!true
!false
exit
