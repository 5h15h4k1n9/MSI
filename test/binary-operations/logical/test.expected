(bool) false
(bool) true
(bool) false
(bool) true
Bye-bye!
